lib/config.rb

```diff
@@ -48,6 +49,14 @@ def self.load_files(*sources)
 
     config.add_source!(Sources::EnvSource.new(ENV)) if Config.use_env
```

Comment:
Let's load credentials _before_ the environment. My expectation is usually that env vars should "win" against every other configuration source.